plugin/pkg/admission/priority/admission.go

```diff
@@ -162,12 +162,15 @@ func (p *PriorityPlugin) admitPod(a admission.Attributes) error {
 			if !ok {
 				// Now that we didn't find any system priority, try resolving by user defined priority classes.
 				pc, err := p.lister.Get(pod.Spec.PriorityClassName)
+
 				if err != nil {
-					return fmt.Errorf("failed to get default priority class %s: %v", pod.Spec.PriorityClassName, err)
-				}
-				if pc == nil {
-					return admission.NewForbidden(a, fmt.Errorf("no PriorityClass with name %v was found", pod.Spec.PriorityClassName))
+					if errors.IsNotFound(err) {
+						return err
```

Comment:
If you do this, when you attempt to do a create, you'll get a 404, which doesn't make sense generically since you're posting a place that exists.

Reply:
It's nice, i have tried to do it again, PTAL, thank you!